tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -592,7 +592,7 @@ public async Task<long> SendObjectsPackFileAsync(string commitSha, string[]? com
         long totalUploadSize = 0;
         foreach (var packFile in packFilesObject.Files)
         {
-            if (!File.Exists(packFile))
+            if (!Directory.Exists(Path.GetDirectoryName(packFile)) || !File.Exists(packFile))
```

Comment:
Does this actually do anything extra? 🤔

Reply:
To be honest I don't think so, but I mean... I'm already doing a File.Exist but for some reason later is throwing DirectoryNotFoundException, so I don't know if is something weird on debian+arm64 pal implementation or what. Adding this doesn't hurt but I'm with you, I don't think it will be relevant 🤷🏻